src/OpenTelemetry/CHANGELOG.md

```diff
@@ -19,6 +19,11 @@
   Previously it would return `null`.
   ([#4885](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4885))
 
+* Updated `Microsoft.Extensions.Logging` package version to
```

Comment:
nit: I think we can mention that this is now a direct dep, vs trasitively coming via Ms.Ext.Log.Config..

Reply:
How much value would this give to the user?